src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -423,42 +423,42 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                 }
             }
 
+            let populateNearInteractionInfo = (nearInteractionInfo: Nullable<PickingInfo>, nearHover: boolean): Nullable<PickingInfo> =>{
+                let result = null;
+                let nearInteractionAtOrigin = false;
+                let nearInteraction = nearInteractionInfo && nearInteractionInfo.pickedPoint && nearInteractionInfo.hit;
+                if (nearInteractionInfo ?.pickedPoint) {

```

Comment:
Does this work with a space before the '?' ? I learned something new...

Reply:
this code is tested and it works, so i suppose yes